app.py

```diff
@@ -1,28 +1,22 @@
 from typing import Dict
 
-import streamlit as st
-import wikipedia
-from transformers import Pipeline
-from transformers import pipeline
+import streamlit as st  # type: ignore
+import wikipedia    # type: ignore
+from transformers import Pipeline, pipeline # type: ignore
 
 NUM_SENT = 10
 
 
-@st.cache
 def get_qa_pipeline() -> Pipeline:
-    qa = pipeline('question-answering')
+    qa = pipeline("question-answering", framework="tf")
```

Comment:
Can we not set this to use TF by default? Since I tested this using Pytorch, I didn't run into the Streamlit caching issue. One option is we can pass the framework (i.e. tf/pytorch) as a command line argument and then have a cached/non-cached `get_qa_pipeline` method. We can then call the correct one based on the user's inputs.

Reply:
I reran again using with pytorch 1.6, yet I face the same issue so have removed caching